tianshou/trainer/base.py

```diff
@@ -426,64 +408,98 @@ def test_step(self) -> tuple[CollectStats, bool]:
 
         return test_stat, stop_fn_flag
 
-    def train_step(self) -> tuple[CollectStats, bool]:
-        """Perform one training step.
+    @contextmanager
+    def _is_training_step_enabled(self, is_training_step: bool):
```

Comment:
see my comment above - rename to `_in_training_rollout`?

Reply:
See comment above.